### PR TITLE
Handle org resources specially

### PR DESCRIPTION
**Pull request by kkaneda**
We cannot handle the access control of orgs and projects with the current mechanism, which only works when a resource is project-scoped.

For example, when a user attempts to list all projects in a given org, we need to let a caller know which project is visible. On the other hand, the current authorization API checks if a user is allowed to read/write to a resource in a specific project and org.

We could extend the API, but it is simpler to implement the auth logic in the caller (= user-manager).

### CODE REVIEW

File: configs/server/config.yaml
Comment (Ladicle) [inline configs/server/config.yaml:1]: Is this file intended for a local test?

**Review by Ladicle — APPROVED** [review]

File: configs/server/config.yaml
Comment (kkaneda) [reply configs/server/config.yaml:1]: Ah let me remove.
